exporter/exporterhelper/tracehelper_test.go

```diff
@@ -39,15 +40,13 @@ const (
 // TODO https://github.com/open-telemetry/opentelemetry-service/issues/266
 // Migrate tests to use testify/assert instead of t.Fatal pattern.
 func TestTraceExporter_InvalidName(t *testing.T) {
-	if _, err := NewTraceExporter("", newPushTraceData(0, nil)); err != errEmptyExporterName {
-		t.Fatalf("NewTraceExporter returns: Want %v Got %v", errEmptyExporterName, err)
-	}
+	_, err := NewTraceExporter("", newPushTraceData(0, nil))
```

Comment:
name a variable for the first return and assert it is nil.

Reply:
besides the refactor this becomes an improvement in the test, the same apply to other locations below.